### PR TITLE
#35 maintenance api needs endpoint for component maint

### PR DESCRIPTION
**Pull request by MS-ResilienX**
Related to functionality introduced to FRAIHMWORK on FRAIHM-1772.

### CODE REVIEW

**Review by GD-ResilienX — APPROVED** [review]
Looks good 👍

**Review by AS-resilienx — APPROVED** [review]

File: MaintenanceApi.yaml [hunk 1]
```diff
@@ -384,6 +490,72 @@ components:
...
+          description: This is the amount of time that the component has been AVAILABLE since it has been registered with FRAIHMWORK. 
+          type: number
+          format: double
+          
+        totalDownHours:
+          description: This is the amount of time that the component has been DOWN since it has been registered with FRAIHMWORK. 
+          type: number
+          format: double
+
+        totalDownEntries:
```
Comment (GD-ResilienX) [inline MaintenanceApi.yaml:548]: Should be renamed to fit in line with the FRAIHM-1772 requirements, IE, something like "totalNumberOfFailures"

File: MaintenanceApi.yaml [hunk 2]
```diff
@@ -384,6 +490,72 @@ components:
...
+          description: This is the number of times the component has transitioned out of DOWN state due to clearing all repairable faults. 
+          type: integer
+          format: int64
+
+        totalAvailableEntries:
```
Comment (GD-ResilienX) [inline MaintenanceApi.yaml:553]: Should be renamed to fit in line with the FRAIHM-1772 requirements, IE, something like "totalNumberOfRecoveries"

**Review by GD-ResilienX — CHANGES_REQUESTED** [review]

File: MaintenanceApi.yaml (hunk 1 above)
Comment (MS-ResilienX) [reply MaintenanceApi.yaml:548]: Agreed. That would sound better. Will change.

**Review by GD-ResilienX — APPROVED** [review]

**Pull request merged**